### PR TITLE
Windows + Appium integration document changes

### PR DESCRIPTION
**Pull request by t-mamith**
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [ ] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules
## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by triager** [comment]
Can one of the admins verify this patch?

File: docs/en/about-appium/intro.md [hunk 1]
```diff
@@ -2,15 +2,15 @@
 
 Appium is an open-source tool for automating native,
 mobile web, and hybrid applications on iOS and Android platforms.
-**Native apps** are those written using the iOS or Android SDKs. **Mobile web apps** are web apps accessed using a mobile browser (Appium supports
+**Native apps** are those written using the iOS, Windows or Android SDKs. **Mobile web apps** are web apps accessed using a mobile browser (Appium supports
```
Comment (imurchie) [inline docs/en/about-appium/intro.md]: I think we ought to be consistent in the ordering of these references. `iOS, Android, and Windows` makes the most sense for me.

File: docs/en/appium-setup/running-on-windows.md [hunk 2]
```diff
@@ -66,3 +47,29 @@ arguments.
...
+   8. Install [cURL](http://curl.haxx.se/download.html)
+   9. Install  appium-doctor  in order to check your system: npm install appium-doctor -g. Then run by typing  appium-doctor. You should see that all of the tests have passed
+   10. Within test script, change value of 'platformVersion' to correspond with emulator version (i.e. '4.4.4')
+   11. Within test script, change value of 'app' to PATH('path of .apk') and ensure the entire path (including close parenthesis) is on one line
+
+Notes for tests: 
+- URL endpoint has to append "/wd/hub"
+- DesiredCapabilities:
+   - platformName = "Windows"
+   - deviceName = "WindowsPC"
```
Comment (imurchie) [inline docs/en/appium-setup/running-on-windows.md]: This seems wrong for Android.

File: docs/en/appium-setup/running-on-windows.md [hunk 3]
```diff
@@ -1,46 +1,27 @@
...
 To get started:
 
-1. Install [node.js](http://nodejs.org/download/) (v4 or greater). Use the
-   installer from nodejs.org.
-1. Install `appium-doctor` in order to check your system: `npm install appium-doctor -g`.
-   Then run by typing `appium-doctor`.
-1. Install the [Android SDK](http://developer.android.com/sdk/index.html).
-   You will need to run the 'android' tool (included in the SDK, in the `tools` folder) and make sure
-   you have an API Level 17 or greater API installed. Set `ANDROID_HOME` to be
+   1. Download latest [node and npm tools](https://nodejs.org/download/release/v6.3.0/node-v6.3.0-x64.msi) MSI (version >= 6.0). The `npm` and `nodejs` paths should be in your PATH environment variable.
```
Comment (jlipps) [inline docs/en/appium-setup/running-on-windows.md]: ok, this section isn't exactly what I had in mind. Here's what I was thinking:

```md
### Setup

<put shared setup instructions here>

### Android Setup

<put android-ONLY setup instructions here. this is where the java stuff goes>

### Windows App Testing setup

<put winappdriver-ONLY setup instructions here (like developer mode)>
```

In other words, there's no need to duplicate shared instructions.

**Comment by jlipps** [comment]
@t-mamith made a few comments. in addition:

1. I think we should add the winappdriver-specific capability to `caps.md`
2. there shouldn't be a merge commit in this PR; it should just be one atomic commit. you'll need to squash!

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by t-mamith** [comment]
addressed all comments
not sure why this got closed
moved pr to here: https://github.com/appium/appium/pull/6742

**Pull request closed without merging**